example/lib/main.dart

```diff
@@ -362,6 +370,30 @@ class _MyAppState extends State<MyApp> {
     child: Text('Unsubscribe'),
   );
 
+  int typeCounter = 0;
+  int realtimePublishCounter = 0;
+  Widget createChannelPublishButton() => FlatButton(
+    onPressed: (_realtimeChannelState==ably.ChannelState.attached)?() async {
+      print('Sendimg rest message...');
+      dynamic data = messagesToPublish[(realtimePublishCounter++ % messagesToPublish.length)];
+      ably.Message m = ably.Message()..data=data..name='Hello';
+      if(typeCounter%3 == 0){
```

Comment:
I commented on this in another PR.
Why are these same changes in different PRs?

Reply:
In the next PR this went inside a try-catch as a new issue surfaced.